static_src/components/panel_actions.jsx

```diff
@@ -18,16 +14,10 @@ const defaultProps = {
 };
 
 export default class PanelActions extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.styler = createStyler(style);
-  }
-
   render() {
     const alignClass = `panel-actions-${this.props.align}`;
     return (
-      <span className={ this.styler('panel-actions', alignClass) }>
+      <span className={ `panel-actions ${alignClass}` }>
```

Comment:
Maybe remove the `alignClass` tmp var and just put `panel-actions-${this.props.align}` straight in the string.